commune_app/models.py

```diff
@@ -1,3 +1,13 @@
-# from django.db import models
+from django.db import models
+from django.contrib.auth.models import AbstractUser
 
-# Create your models here.
+
+class User(AbstractUser):
+    id = models.IntegerField(unique=True, primary_key=True)
+    first_name = models.CharField(max_length=250)
```

Comment:
IMHO 250 chars for a first name or last name is too long 
```suggestion
    first_name = models.CharField(max_length=50)
```
50 should be long enough